src/functions-and-arrays.js

```diff
@@ -1,19 +1,35 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 < num2) return num2
+   else 
+   return num1
+  }
```

Comment:
If statements work without curly braces in case the body is just one line. For an if-else statement, the best practice is to use curly brackets. This can help avoid unexpected errors where the code behaves differently than what you expect.